service-base/src/main/java/org/eclipse/hono/service/credentials/CompleteCredentialsService.java

```diff
@@ -26,6 +26,7 @@
  *
  * @see <a href="https://github.com/eclipse/hono/blob/1.0-M4/site/content/api/Credentials-API.md">Credentials API</a>
  */
+@Deprecated
```

Comment:
I think adding a `@deprecated` JavaDoc indicating what to use instead would be helpful